lib/std/dynamic_library.zig

```diff
@@ -339,9 +339,14 @@ pub const WindowsDynLib = struct {
     }
 
     pub fn openW(path_w: [*:0]const u16) !WindowsDynLib {
+        const stripped_path = if (mem.len(path_w) >= 4) blk: {
+            const prefix= &[_]u16{ '\\', '?', '?', '\\' };
```

Comment:
This is very nitpicky, but could you insert a space before the `=` so that it looks a bit nicer?

Reply:
Better yet: run `zig fmt` on the codr

Reply:
No problem with that, I will also remove the length check and use startsWith instead